### ISSUE TITLE
First Issue - HowToStart

### ISSUE THREAD

**Issue by WK-GiHu** [Justinjay282/qwerty#1]
Notify me with: @WK-GiHu

- [Stackoverflow - how-to-call-and-close-a-virtual-keyboard-made-by-tkinter-using-touchscreen-displ](https://stackoverflow.com/questions/60136473)

### To **review** your code, update allways the files here after edit.

Start allways at the `[Issues]` page, i open for every single issue a **New Issue**.  First Issue - How to start

**Comment by Justinjay282** [comment]
You are sir stovfl right?

**Comment by WK-GiHu** [comment]
Yes, see [adding-a-file-to-a-repository](https://help.github.com/en/github/managing-files-in-a-repository/adding-a-file-to-a-repository)

**Comment by Justinjay282** [comment]
Thank you so much for doing all this help just to help me. I really appreciate what your doing, I'm sorry if I'm quite dumb in programming but I will do my best with your help.

**Comment by Justinjay282** [comment]
sir @WK-GiHu, I declared another `toplevel` window, `var = Toplevel()` and then `VKeyboard(var)` is it able? I did it but It's not working, sorry for an early question.
